man/docker-load.1.md

```diff
@@ -43,3 +43,4 @@ Restores both images and tags.
 April 2014, Originally compiled by William Henry (whenry at redhat dot com)
 based on docker.com source material and internal work.
 June 2014, updated by Sven Dowideit <SvenDowideit@home.org.au>
+July 2015 update by Mary Anthony <mary@docker.com>
```

Comment:
I never check if these are updated during review, and see them as a nice-to-have. Do you know if there's a (technical) reason for keeping this list, or should we remove them everywhere (because they are really not kept up-to-date)